### DISCUSSION TITLE
No CubeSatSim Band Pass Filter detected

### DISCUSSION THREAD

**Discussion by R3DKR0SS** [alanbjohnston/CubeSatSim#321] [Q&A] [answered]
I've just built the cubesatsim v1.2 and for some reason I'm getting the following error message when running CubeSatSim/cubesatsim:
```
...
No CubeSatSim Band Pass Filter detected.  No transmissions after the CW ID. See http://cubesatsim.org/wiki for info about building a CubeSatSim
...
```
`**_To fix this add a 1k ohm resistor to R1._**`
And I'm sure I soldered the low pass filter to it.
![IMG_3345](https://github.com/alanbjohnston/CubeSatSim/assets/43837752/6e09d964-c76a-4d40-8dc0-426c5ec312fd)

**Comment by alanbjohnston** [comment, marked as answer]
Hi,

You see that message in the log (CubeSatSim/log command) when resistor R1 is not present. On your radio or SDR you will hear the CW ID at the start then nothing. 

Double check that resistor or that pin on the connector J1.

Good luck!

Alan

**Reply by R3DKR0SS** [reply]
Greetings,

Pf how many ohms does that resistor needs to be? 1k?

THE ANSWER WAS 1K OHM

Waiting for your response,
Equis

**Comment by alanbjohnston** [comment]
1k Ohm is the value, but less than that including a short circuit will also work.
